docs/adr/0005-api-for-spaces.md

```diff
@@ -0,0 +1,129 @@
+# 5. Graph API for Spaces
+
+* Status: proposed
+* Deciders: @butonic, @micbar, @dragotin, @hodyroff, @pmaier1
+* Date: 2021-03-19
+
+Technical Story: API to enable the concept of [Spaces](https://github.com/owncloud/enterprise/issues/3863)
+
+## Context and Problem Statement
+
+As one of the building blocks for Spaces in oCIS we plan to add an API that returns information about available spaces. This ADR discusses the API design oriented on the Microsoft Graph API.
+
+> Note: The term "spaces" is used here in the context of "a space where files can be saved", similar to a directory. It is not to be confused with space in the sense of free file space for example.
+
+The purpose of this new API is to give clients a very simple way to query the dynamic list of spaces, that the user has access to. Clients can provide a better user experience with that.
+
+This API is supposed to be queried often, to give clients a condensed view of the available spaces for a user, but also their eTags and cTags. Hence the clients do not have to perform a PROPFIND for every space separately.
```

Comment:
Please clarify what cTag is and how it differs from a ETAG?

Reply:
CTag indicates that the content of the file has changed.
ETag indicates that content or metadata has changed.

Reply:
I agree with @labkode, a section with vocabulary would help